packages/dataviews/src/view-list.tsx

```diff
@@ -142,23 +205,137 @@ function ListItem( {
 						</HStack>
 					</CompositeItem>
 				</div>
+				{ actions?.length > 0 && (
+					<HStack
+						spacing={ 1 }
+						justify="flex-end"
+						className="dataviews-item-actions"
```

Comment:
Also we shouldn't reuse classnames in our code base.

Reply:
Good catch, thanks. Fixed at https://github.com/WordPress/gutenberg/pull/60805/commits/7d76a8e1baf1351e1b2b2524da2a68682b93866b